modules/solid_mechanics/doc/content/source/materials/CappedWeakPlaneStressUpdate.md

```diff
@@ -1 +1,767 @@
-!template load file=stubs/moose_object.md.template name=CappedWeakPlaneStressUpdate syntax=/Materials/CappedWeakPlaneStressUpdate
\ No newline at end of file
+# CappedWeakPlaneStressUpdate
+
+!syntax description /Materials/CappedWeakPlaneStressUpdate
+
+## Theory id=theory
+
+Weak-plane plasticity is designed to simulate a layered material.
+Each layer can slip over adjacent layers, and be separated from those
+adjacent layers.  An example of particular interest is
+stratified rocks, which consist of large sheets of fairly strong rock
+separated by weak and very thin joints.  Upon application of stress,
+the joints can fail, either by slipping or separating.  The idea is to
+use one finite element that potentially contains many layers, and
+prescribe ``weak plane plasticity'' for that finite element, so that
+it can fail by joint separation and joint slipping.
+
+Denote the normal to the layers by $z$, and the tangential directions
+by $x$ and $y$.  It is convenient to introduce two new stress variables in
+terms of the stress tensor $\sigma$:
+
+\begin{equation}
+p = \sigma_{zz} \ \ \ \mbox{and}\ \ \ q = \sqrt{\sigma_{xz}^{2} +
+  \sigma_{yz}^{2}} \ .
+\label{eqn.defn.p.q}
+\end{equation}
```

Comment:
This is not rendering correctly because I don't think `\mbox` is supported in the .md equations. It works just fine in .tex.